app/src/panel/models/page/blueprint/pages.php

```diff
@@ -22,7 +22,7 @@ class Pages extends Obj {
   public function __construct($params = array()) {
 
     if($params === true) {
-      $this->template = blueprint::all();
+      $this->template = true;
```

Comment:
Well at least on a broader scale `pages: true` always implies `templates: all`... Maybe @bastianallgeier knows a nicer way to still get the complete list from `blueprint:all()` without delaying it to lines 54-56.